docs/journalist/closing_client.rst

```diff
@@ -2,3 +2,7 @@ Closing the SecureDrop client
 =============================
 
 .. include:: ../includes/top-warning.rst
+
+When you are finished checking SecureDrop, you can sign out by clicking your username in the top lefthand panel, and clicking **Sign out.**
```

Comment:
sounds good. changed in 1447bdd as an either/or

Reply:
It's not an either/or tho. Closing the window kills the client. Signing out leaves it running in offline mode.

Reply:
Good point! I guess in my mind we were also telling users to immediately power off the machine too as the next step. I will leave it as 'close the window' and remove the signout part.

Reply:
fixed in 1ef26a8

Reply:
"Power off" is handled in the next section "Shutting down Qubes OS", and "power off" could be interpreted as "press the power button". Perhaps these two sections should be merged, since they're both very short?

Reply:
combined in 5eda33f in a section called "Ending your session"

Reply:
Merge makes sense!